README.md

```diff
@@ -1,7 +1,10 @@
 # Chat-App
```

Comment:
Remove all this.. 

and this code.. copy from below link and just paste it.
https://www.codelog.tk/3hn